Apps/WebClient/src/WebClient.csproj

```diff
@@ -87,7 +93,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="$(SpaRoot)dist\**" />
-      <ResolvedFileToPublish Include="@(DistFiles-&gt;'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
+      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```

Comment:
I think this should be good, but just in case, can you make sure?

Reply:
This is a VS Mac vs Windows encoding issue... its a pain in the butt as you can't reset the file if you make any changes which I did in this PR and manually setting it back is annoying.

Reply:
I found this: 
https://developercommunity.visualstudio.com/content/problem/861667/-to-gt-in-cproj-file.html?inRegister=true

No solution identified though.